commands_to_load/HealthChecks.py

```diff
@@ -3,13 +3,15 @@
 import discord.client
 import json
 from commands_to_load.Paginate import paginateEmbed, paginate
+from helper_files.embed import embed 
+import helper_files.settings as settings
 
 import logging
 logger = logging.getLogger('wall_e')
 
 class HealthChecks():
 
-	def __init__(self, bot):
+	def __init__(self, bot:commands.bot.Bot):

```

Comment:
this was left in by accident. doesnt do anything in the code, but intellij(i think thats wats in vscode) then can show all methods and properties since it knows what object that is.